cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/visitor/BaseKnowledgeArtifactVisitor.java

```diff
@@ -34,11 +34,17 @@
 public abstract class BaseKnowledgeArtifactVisitor implements IKnowledgeArtifactVisitor {
     String isOwnedUrl = "http://hl7.org/fhir/StructureDefinition/artifact-isOwned";
     protected final Repository repository;
+    protected Optional<IValueSetExpansionCache> valueSetExpansionCache = Optional.empty();
 
     protected BaseKnowledgeArtifactVisitor(Repository repository) {
         this.repository = repository;
     }
```

Comment:
```suggestion
    protected final Optional<IValueSetExpansionCache> valueSetExpansionCache;

    protected BaseKnowledgeArtifactVisitor(Repository repository) {
        this.repository = repository;
        this.valueSetExpansionCache = Optional.empty();
    }
```

Reply:
Or just get rid of this constructor.  The children can still have a constructor without a cache.